xbmc/guilib/FFmpegImage.cpp

```diff
@@ -225,7 +225,7 @@ bool CFFmpegImage::Decode(unsigned char * const pixels, unsigned int width, unsi
   if (m_width == 0 || m_height == 0 || format != XB_FMT_A8R8G8B8)
     return false;
 
-  if (!m_pFrame || !m_pFrame->data)
+  if (!m_pFrame || !m_pFrame->data[0])
```

Comment:
```
 CID 1343330:    (NO_EFFECT)
>>>     Comparing an array to null is not useful: "!this->m_pFrame->data", since the test will always evaluate as true.
228       if (!m_pFrame || !m_pFrame->data)
```

That's not a simple ptr, it's ptr* to something[something_else]